joomla/joomla-cms
# [new feature] Article notes field

We have a note field for com_categories, com_menus and com_modules
This PR adds the functionality to com_content as it has been requested several times and can be very useful

## To test
Apply the pr AND the database fix (we added a new field)
Create an article and on the right, above version note you will see a new field called note
Enter a note and save the article
You can now see the note with the article in the list
You can also search by the note
Check the featured article list
Check the article modal list (article xtd-editor button or from creating a new single article menu)

On the frontend you can also add/edit a note. You will see it on the publishing tab above the version note field

### Bonus
When adding this field to the frontend beez template I spotted that the version note field wasnt shown there either so I added that at the same time
